onboard/client/client.py

```diff
@@ -91,6 +91,15 @@ def get_all_points(self) -> List[int]:
             for e in equipment:
                 point_ids += e['points']
         return point_ids
+    
+    def get_all_equipment(self) -> List[int]:
```

Comment:
the type annotation here doesn't look correct

Reply:
Should it be List[dict] ?
I used the same type annotation as in the method " get_all_points(self) -> List[int] ", which returns a similar output (a list of dictionaries).

Reply:
I can fix the type annotation of get_all_points too if that one should be List[Dict]

Reply:
ah yeah that'd be great - thank you. I'm going to file a ticket to add automated checks for these, too